experimental/PropertyDDS/packages/property-inspector-table/src/ExpiryModal.tsx

```diff
@@ -195,12 +195,12 @@ export const ExpiryModal: React.FunctionComponent<IExpiryModalProps> = (props) =
    * Set the expiry of the current repository to the provided retention policy
    * @param expiryTime The new retention policy/expiry time.
    */
-  const setExpiry = (expiryTime: ExpiryTimeType): Promise<void> => {
+  const setExpiry = async (expiryTime: ExpiryTimeType): Promise<void> => {
     setPolicyState({ ...policyState, updating: true });
     setModalExpiryState({ ...modalExpiryState, expiresIn: expiryPlaceHolder });
     return ErrorPopup(setRepoExpiry.bind(null, repositoryUrn!, expiryTime)).then(() => {
-      setModalState({ mode: 'default' });
-      getExpiryInfo().then((expiryInfo) => {
+      setModalState({ mode: "default" });
+      void getExpiryInfo().then((expiryInfo) => {
```

Comment:
Note to self: unhandled promise

Reply:
LMK if you would prefer eslint-disables for these.

Reply:
I switched the voids to eslint-disables in the latest iteration -- if you do decide to go after them it's easier to find them later.